packages/labextension/src/widget.tsx

```diff
@@ -0,0 +1,268 @@
+        callback: createKernelChangeCallback(panel),
+        panel,
+      };
+      panel.sessionContext.kernelChanged.connect(kernelChangeCallback.callback);
+
+      if (panel.sessionContext.session?.kernel?.id !== kernelId) {
+        const kernelId = panel.sessionContext.session?.kernel?.id;
+        if (kernelId) {
+          setKernelId(kernelId);
+          const path = panel.sessionContext.session?.model.path;
+          setPath(path);
+          requestUsage(kernelId).then((usage) => setUsage(usage));
+        }
+      }
+    };
+    props.currentNotebookChanged.connect(notebookChangeCallback);
+    return () => {
+      props.currentNotebookChanged.disconnect(notebookChangeCallback);
+      // In the ideal world we would disconnect kernelChangeCallback from
+      // last panel here, but this can lead to a race condition. Instead,
+      // we make sure there is ever only one callback active by holding
+      // it in a global state.
+    };
+  }, [kernelId]);
+
+  if (kernelId) {
+    if (usage) {
+      return !usage.hostname ? (
+        <>
+          <h3 className="jp-KernelUsage-section-separator">
+            {props.trans.__('Kernel usage details are not available')}
+          </h3>
+          <div className="jp-KernelUsage-section-separator">
+            {props.trans.__(
+              'Please check with your system administrator that you running IPyKernel version 6.10.0 or above.'
+            )}
+          </div>
+        </>
+      ) : (
+        <>
+          <h3 className="jp-KernelUsage-section-separator">
+            {props.trans.__('Kernel usage')}
+          </h3>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Kernel Host')}: {usage.hostname}
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Notebook')}: {path}
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Kernel ID')}: {kernelId}
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Timestamp')}: {usage.timestamp?.toLocaleString()}
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Process ID')}: {usage.pid}
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('CPU')}: {usage.kernel_cpu}% used
+          </div>
+          <div className="jp-KernelUsage-separator">
+            {props.trans.__('Memory')}: {formatForDisplay(usage.kernel_memory)}
+          </div>
+          <hr className="jp-KernelUsage-section-separator"></hr>
+          <h4 className="jp-KernelUsage-section-separator">
+            {props.trans.__('Host CPU')}
+          </h4>
+          {usage.host_cpu_percent && (
+            <div className="jp-KernelUsage-separator">
+              {usage.host_cpu_percent.toFixed(1)}% {props.trans.__('used on')}{' '}
+              {usage.cpu_count} CPUs
```

Comment:
Something like:

```suggestion
              {props.trans._n('%2%% used on %1 CPU', '%1%% used on %2 CPUs', usage.cpu_count, usage.host_cpu_percent.toFixed(1))}
```

Reply:
Do you mean the following?
```
{props.trans._n('%2%% used on %1 CPU', '%2%% used on %1 CPUs', usage.cpu_count, usage.host_cpu_percent.toFixed(1))}
```

Reply:
Yes, thanks for catching, I updated the comment.

Reply:
Ah, already pushed in https://github.com/jupyter-server/jupyter-resource-usage/pull/164/commits/1a24212b916beff4e15ca49085a63358a62901b5.